Tools/CISupport/ews-build/steps.py

```diff
@@ -4944,13 +4958,27 @@ def evaluateCommand(self, cmd):
         rc = super(UpdatePullRequest, self).evaluateCommand(cmd)
 
         loglines = self.log_observer.getStdout().splitlines()
+        bug_id = None
 
         title = loglines[4][4:].rstrip()
         description = f'#### {loglines[0].split()[1]}\n<pre>\n'
         for line in loglines[4:]:
+            if not bug_id:
+                for word in line.split():
+                    for candidate in self.BUGS_RE:
+                        match = candidate.match(word)
+                        if match:
+                            bug_id = match.group('id')
+                            break
+                    if bug_id:
+                        break
+
             description += self.escape_html(line[4:] + '\n')
         description += '</pre>\n'
 
+        if bug_id:
+            self.setProperty('bug_id', bug_id, 'commit')
```

Comment:
Can have more descriptive name than 'commit', maybe the step name, or can just leave it to default ( i guess default is the step name).